tests/tasks/test_juniperalarmtask.py

```diff
@@ -77,7 +77,26 @@ async def test_task_creates_alarm_events(self, snmpsim, juniper_alarm_task):
 
         assert yellow_event
         assert red_event
-        assert yellow_event.alarm_count == 0
+        assert yellow_event.alarm_count == 1
+        assert red_event.alarm_count == 2
+
+    @pytest.mark.asyncio
+    async def test_task_creates_one_alarm_event_on_one_count_changed(self, snmpsim, juniper_alarm_task):
```

Comment:
A hint: If you do not actually need to reference the snmpsim fixture content directly from your test, you don't need to add it to your test signature: `juniper_alarm_task` already depends on the `snmpsim` fixture, so it will already be present.  That's the beauty of fixtures being able to depend on other fixtures.